### PR TITLE
cri: Add support for cri-contaienrd pod annotations

### PR DESCRIPTION
**Pull request by jcvenegas**
Sandbox and Container annotations are now provided by cri-contianerd
this commit add support for this new annotations.

Fixes: #569

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

**Comment by sameo** [comment]
cc @miaoyq @guangxuli @xiangpengzhao @yanxuean

**Comment by sameo** [comment]
LGTM

File: pkg/oci/utils.go [hunk 1]
```diff
@@ -23,24 +23,37 @@ import (
...
 	// the container type from annotations in the config.json.
-	CRIContainerTypeKeyList = []string{annotations.ContainerType}
+	CRIContainerTypeKeyList = []string{criContainerdAnnotations.ContainerType, crioAnnotations.ContainerType}
 
 	// CRISandboxNameKeyList lists all the CRI keys that could define
 	// the sandbox ID (pod ID) from annotations in the config.json.
-	CRISandboxNameKeyList = []string{annotations.SandboxID}
+	CRISandboxNameKeyList = []string{criContainerdAnnotations.SandboxID, crioAnnotations.SandboxID}
+
+	CRIContainerTypeList = []annotationContainerType{
```
Comment (sameo) [inline pkg/oci/utils.go]: You need to add a comment for that one, as it's an exported variable.

File: pkg/oci/utils.go [hunk 2]
```diff
@@ -23,24 +23,37 @@ import (
 	"strconv"
 	"strings"
 
+	criContainerdAnnotations "github.com/containerd/cri-containerd/pkg/annotations"
```
Comment (jodh-intel) [inline pkg/oci/utils.go]: Doesn't this need to be vendored?

File: pkg/oci/utils.go (hunk 1 above)
Comment (jcvenegas) [reply pkg/oci/utils.go]: Comment added.

**Comment by sameo** [comment]
@jodh Does that look ok to you ?

File: pkg/oci/utils.go (hunk 2 above)
Comment (jodh-intel) [reply pkg/oci/utils.go]: Ta.

**Comment by jodh-intel** [comment]
lgtm

**Pull request merged**